lib/tasks/migration.rake

```diff
@@ -74,15 +74,17 @@ namespace :migration do
   end
 
   desc "batch migrate generic files from modified ERA FOXML file"
-  task :eraitem, [:dir, :migrate_datastreams] => :environment do |t, args|
+  task :eraitem, [:dir, :migrate_datastreams, :usecomm, :usecoll] => :environment do |t, args|
```

Comment:
Would `usecom`/`usecoll` just be a string containing the noid for the community/collection here? Or are they just booleans? Seems like they contain the noid as you are pushing whatever is passed in as args onto the communities/collection noid array below on lines 470-480ish? 

So a very small nitpick might be to maybe name these args different? The verb `use` implies a boolean, aka yes/no in whether you should use comm/coll etc? Maybye `community_noid`/`collection_noid` might be a better name? Additonally a comment with an example of how to use this rake task wouldn't be that bad of idea either? 
Like `# rake migration:eraitem[code/example/directory/, true, comm_noid-123456, coll_noid-123456]` 

Other then that, this code looks good from a quick glance! Feel free to pull it in whenever you want! :shipit: